src/components/todo/all-list-todo.tsx

```diff
@@ -115,25 +73,37 @@ export default function AllListTodo({
       if (filter === "done") return todo.isDone === true;
       if (filter === "todo") return todo.isDone === false;
       return true;
-    })
-    .slice(0, maxItems);
+    });
 
   return (
-    <div className="mx-auto min-h-[2080] w-full max-w-2xl rounded-xl rounded-b-none border-slate-300 bg-white p-20 text-sm text-slate-800">
-      <Filter filter={filter} setFilter={setFilter} />
-      <ul className="flex flex-col gap-20">
-        {todos.map((todo, index) => (
-          <ListTodo
-            key={todo.id}
-            index={index}
-            todo={todo}
-            toggleStatus={toggleStatus}
-            showDropdownOnHover // 드롭다운 호버 여부
-            showGoal // 목표 보여주기 여부
-          />
-        ))}
-      </ul>
-      <div ref={ref} className="h-1" />
-    </div>
+    <>
+      <div className="mb-16 mt-24 flex items-center justify-between">
+        <h2 className="text-18 font-semibold text-slate-600">{`모든 할 일 (${todos.length})`}</h2>
+        <button
+          onClick={() => setIsModalOpen(true)}
+          className="flex items-center gap-3 font-semibold text-dark-blue-600"
+        >
+          <PlusIcon />
+          <span className="text-14/3">할일 추가</span>
+        </button>
+      </div>
+      <div className="min-h-[2080px] w-full rounded-xl rounded-b-none border-slate-300 bg-white p-20 text-sm text-slate-800">
+        <Filter filter={filter} setFilter={setFilter} />
```

Comment:
요 부분 저번에 한번 얘기 했었던 것 같은데 맞을까요? pxToRem 쓰니 다른 코드에서처럼  수정하는 것이 일관성이 있을 것 같아요!

```suggestion
      <div className="min-h-2080 w-full rounded-xl rounded-b-none border-slate-300 bg-white p-20 text-sm text-slate-800">
        <Filter filter={filter} setFilter={setFilter} />
```

Reply:
@ChaeYubin @hvrain  이거는 제가 min-h-2080 이렇게 쓰니까 적용이 안되는 것을 확인했었는데 말씀을 안 드린 것 같네요.

Reply:
오호.. 네 알겠습니다!